src/backend/looped/self/beast_jump.cpp

```diff
@@ -10,12 +8,7 @@ namespace big
 		using looped_command::looped_command;
 
 		virtual void on_tick() override
-		{
-			if (PAD::IS_CONTROL_JUST_PRESSED(0, (int)ControllerInputs::INPUT_JUMP) && !PED::IS_PED_IN_ANY_VEHICLE(self::ped, NULL) && !ENTITY::IS_ENTITY_IN_AIR(self::ped))
-			{
-				TASK::TASK_JUMP(self::ped, true, true, true);
-			}
-		}
+		{}
 	};
 
 	beast_jump_looped g_beast_jump_looped("beastjump", "Beast Jump", "Allows you to jump as if you were the beast like in the Hunt the Beast event",
```

Comment:
Is any of this still used? If all you're using is the boolean, you can just register a `bool_command`.

Example:
https://github.com/YimMenu/YimMenu/blob/10bbf1f4bb93d8e6c82c7da98151166f00a07d68/src/backend/looped/self/godmode.cpp#L40